src/ImageCore.jl

```diff
@@ -4,6 +4,7 @@ module ImageCore
 
 using Colors, FixedPointNumbers, MappedArrays, Graphics, ShowItLikeYouBuildIt
 using OffsetArrays # for show.jl
+using ColorTypes: colorant_string
```

Comment:
observant, picky, OCD perhaps

this direct dependency stuff will hopefully be baked into the way Pkg3 environments work so it'll get auto-detected